lib/builderator/tasks/vagrant.rb

```diff
@@ -81,8 +85,8 @@ def ssh(profile = :default, *args)
           command = Interface.vagrant.command
           command << " ssh #{args.join(' ')}"
 
-          ## Connect to subprocesses STDIO
-          exec(command)
+          return run command if Interface.vagrant.bundled?
```

Comment:
Since we're using the `return` keyword and `if` modifier here how about using parens for clarity?

```ruby
# okay - but arguably unclear
return run command if Interface.vagrant.bundled?

# good - parens show precedence
return run(command) if Interface.vagrant.bundled?
```